modules/navier_stokes/include/actions/NSFVAction.h

```diff
@@ -359,12 +361,13 @@ NSFVAction::checkBlockwiseConsistency(const std::string block_param_name,
 
   if (block_names.size())
   {
-    for (const auto & block_group : block_names)
-      for (const auto & block : block_group)
-        if (std::find(_blocks.begin(), _blocks.end(), block) == _blocks.end())
-          paramError(block_param_name,
-                     "Block '" + block +
-                         "' is not present in the block restriction of the fluid flow action!");
+    if (_blocks.size())
```

Comment:
A comment here about why you are adding the new `_blocks.size()` check would be great

Reply:
Added! This also changed a little.